packages/icon/README.md

```diff
@@ -21,8 +13,6 @@ import { Icon } from '@ag.ds-next/icon';
 </Stack>
 ```
 
-## All icons
-
-See the full list of available icons. See something is missing? Get in touch and we'll add it!
+See the full list of available icons [here](https://steelthreads.github.io/agds-next/storybook/index.html?path=/story/foundations-icon--all-icons).
 
-<IconExamples />
+See something is missing? Get in touch and we'll add it!
```

Comment:
not sure if we want an open invite to add stuff especially icons. Can we change to 'do you have a user need for new or missing icon? Get in touch to discuss an icon contribution.'